rpcs3/Emu/RSX/rsx_cache.h

```diff
@@ -710,17 +707,19 @@ namespace rsx
 
 		RSXFragmentProgram load_fp_raw(u64 program_hash)
 		{
-			std::vector<u8> data;
-			std::string filename = fmt::format("%llX.fp", program_hash);
-
-			fs::file f(root_path + "/raw/" + filename);
-			if (f) f.read(data, f.size());
+			fs::file f(fmt::format("%s/raw/%llX.fp", root_path, program_hash));
 
 			RSXFragmentProgram fp = {};
-			fp.ucode_length = ::size32(data);
+
```

Comment:
I'd rather we early return in case of an invalid file tbh, but its just a suggestion. The method isn't overly complex anyway.

Reply:
Fixed.